backend/apps/notifications/channels/registry.py

```
@@ -0,0 +1,48 @@
+    channels: Dict[str, NotificationChannel]
+
+
+def get_channel_registry() -> ChannelRegistry:
+    channel_map = getattr(settings, "NOTIFICATION_CHANNELS", None) or DEFAULT_CHANNELS
+
+    instances: Dict[str, NotificationChannel] = {}
+    for channel_id, dotted in channel_map.items():
+        try:
+            cls = _load_class(dotted)
+            instances[channel_id] = cls()
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)